jetty-websocket/javax-websocket-server-impl/src/main/java/org/eclipse/jetty/websocket/jsr356/server/deploy/WebSocketServerContainerInitializer.java

```diff
@@ -129,54 +132,117 @@ public static boolean isEnabledViaContext(ServletContext context, String keyName
     
         return defValue;
     }
-    
+
     /**
-     * Embedded Jetty approach for non-bytecode scanning.
-     * @param context the {@link ServletContextHandler} to use
-     * @return a configured {@link ServerContainer} instance
-     * @throws ServletException if the {@link WebSocketUpgradeFilter} cannot be configured
+     * Initialize the {@link ServletContext} with the default (and empty) {@link ServerContainer}
+     *
+     * @param context the context to work with
      */
-    public static ServerContainer configureContext(ServletContextHandler context) throws ServletException
+    public static void initialize(ServletContext context) throws ServletException
```

Comment:
javadoc should describe how this is different to `onStartup` or using `configure`.
If there are not real differences, perhaps we just call onStartup?

Reply:
Done.

Reply:
I've been reworking this to be more clear.

so far ... `configure(ServletContextHandler, Configurator)` is used when you want to enable the functionality of the SCI in an embedded jetty scenario.
With an optional `Configurator` to perform actions against that configuration.

.. `initialize(ServletContextHandler)` is always an immediate initialization of the default component for that SCI.  This is used by the SCI's own `onStartup()`, other SCIs, and optionally by Embedded Jetty users.

I'll have a commit for this shortly.